labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,14 +35,19 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val google  = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv y e q"
+    val english = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up a o z"
+    val map = google.zip(english).toMap
+    lines.map(_.map(map.get(_).getOrElse('?')))
```

Comment:
This is a neat solution! However, I think a get would be preferred over a getOrElse to follow the "fail fast" paradigm.